test/app/SIPDNSManagerUnitTest.cs

```diff
@@ -58,5 +58,39 @@ public void LookupLocalHostnameTest()
 
             logger.LogDebug($"resolved to SIP end point {resultEP}");
         }
+
+        [Fact]
+        public void ResolveSIPServiceTest()
+        {
+            logger.LogDebug("--> " + System.Reflection.MethodBase.GetCurrentMethod().Name);
+
+            var result = SIPDNSManager.ResolveSIPService(SIPURI.ParseSIPURIRelaxed("sip:reg.sip-trunk.telekom.de;transport=tcp"), false);

```

Comment:
Is this code reaching out to the internet?

If yes it doesn't seem correct to have a dependency to systems that we don't have any control over.

Reply:
the referenced address is a public SIP-Trunk of THE major communications-provider in western europe. There is a fair chance these addresses stay valid for a long time (if not you always could change the address in test)
Intention of this test is to resolve NAPTR, SRV and A records. Do you know any other public address?.